### PR TITLE
Permalinks: Detect Caddy server

### PR DESCRIPTION
**Pull request by swissspidy**
Trac ticket: https://core.trac.wordpress.org/ticket/41877

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/vars.php [hunk 1]
```diff
@@ -126,6 +126,12 @@
  */
 $is_nginx = ( str_contains( $_SERVER['SERVER_SOFTWARE'], 'nginx' ) );
 
+/**
+ * Whether the server software is Caddy or something else
+ * @global bool $is_caddy
```
Comment (mukeshpanchal27) [inline src/wp-includes/vars.php]:
```suggestion
 * Whether the server software is Caddy or something else
 *
 * @global bool $is_caddy
```

Comment (mukeshpanchal27) [reply src/wp-includes/vars.php]: The end period is missing in all short description.

Comment (swissspidy) [reply src/wp-includes/vars.php]: Better to handle that in a separate commit

**Comment by swissspidy** [comment]
Committed in https://core.trac.wordpress.org/changeset/57612

**Pull request closed without merging**